website/package.json

```diff
@@ -14,8 +14,8 @@
     "write-heading-ids": "docusaurus write-heading-ids"
   },
   "dependencies": {
-    "@docusaurus/core": "2.1.0",
-    "@docusaurus/preset-classic": "2.1.0",
+    "@docusaurus/core": "2.2.0",
+    "@docusaurus/preset-classic": "2.2.0",
     "@mdx-js/react": "^1.6.22",
     "clsx": "^1.1.1",
     "prism-react-renderer": "^1.3.3",
```

Comment:
Based on the current version of Docusaurus 2.2.0 can the following libraries be updated?
-  "@mdx-js/react": "^1.6.22",
-  "clsx": "^1.1.1",
-  "prism-react-renderer": "^1.3.3",
-  "react": "^17.0.2",
-  "react-dom": "^17.0.2",
-  "react-zoom-pan-pinch": "^2.1.3"



Also you will need to run `npm install` and add the newest/updated [package-lock.json](https://github.com/nasa/cumulus-orca/blob/master/website/package-lock.json) file to this PR.

Reply:
- "@mdx-js/react": "^1.6.22", -> 2.1.1
ERROR:
`This page crashed.
(0 , _mdx_js_react__WEBPACK_IMPORTED_MODULE_2__.mdx) is not a function`

- "clsx": "^1.1.1" -> 1.2.1
updated

- "prism-react-renderer": "^1.3.3" -> 1.3.5
updated

- "react": "^17.0.2" -> 18.2.0
`React 16.8.4
npm ERR! Could not resolve dependency:
npm ERR! peer react@"^16.8.4 || ^17.0.0" from @docusaurus/core@2.2.0`

- "react-dom": "^17.0.2" -> 18.2.0
`npm ERR! Could not resolve dependency:
npm ERR! peer react@"^18.2.0" from react-dom@18.2.0
npm ERR! node_modules/react-dom
npm ERR!   react-dom@"^18.2.0" from the root project`

- "react-zoom-pan-pinch": "^2.1.3" 
already has the latest version